plugins/modules/ios_vrf_global.py

```diff
@@ -0,0 +1,845 @@
+                type: bool
+      ipv6:
+        description: VRF IPv6 configuration
+        type: dict
+        suboptions:
+          multicast:
+            description: IP Multicast configuration
+            type: dict
+            suboptions:
+              multitopology:
+                description: Enable Multicast-Specific RPF Topology
+                type: bool
+      rd:
+        description: Specify Route Distinguisher (RD).
+        type: str
+      route_target: *route_target
+      vnet:
+        description: Virtual NETworking configuration.
+        type: dict
+        suboptions:
+          tag:
+            description: Identifier used to tag packets associated with this VNET
+            type: int
+      vpn:
+        description: Configure VPN ID for the VRF as specified in RFC 2685
+        type: dict
+        suboptions:
+          id:
+            description: Configure VPN ID in RFC 2685 format
+            type: str
+  running_config:
+    description:
+      - This option is used only with state I(parsed).
+      - The value of this option should be the output received from the IOS device by
+        executing the command B(show running-config vrf).
+      - The state I(parsed) reads the configuration from C(running_config) option and
+        transforms it into Ansible structured data as per the resource module's argspec
+        and the value is then returned in the I(parsed) key within the result.
+    type: str
+  state:
+    choices: [parsed, gathered, deleted, merged, replaced, rendered, overridden]
+    default: merged
+    description:
+      - The state the configuration should be left in
+      - The states I(rendered), I(gathered) and I(parsed) does not perform any change
+        on the device.
+      - The state I(rendered) will transform the configuration in C(config) option to
+        platform specific CLI commands which will be returned in the I(rendered) key
+        within the result. For state I(rendered) active connection to remote host is
+        not required.
+      - The state I(gathered) will fetch the running configuration from device and transform
+        it into structured data in the format as per the resource module argspec and
+        the value is returned in the I(gathered) key within the result.
+      - The state I(parsed) reads the configuration from C(running_config) option and
+        transforms it into JSON format as per the resource module parameters and the
+        value is returned in the I(parsed) key within the result. The value of C(running_config)
+        option should be the same format as the output of command I(show running-config vrf).
+        connection to remote host is not required.
+    type: str
+"""
+
+EXAMPLES = """
+
+"""
+
+RETURN = """
+before:
+  description: The configuration prior to the module execution.
+  returned: when I(state) is C(merged), C(replaced), C(overridden), C(deleted) or C(purged)
+  type: dict
```

Comment:
as per the design/model, it should be a list of dictionary

Reply:
yes it would be once i add actual examples.